orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/TaskController.groovy

```diff
@@ -330,6 +332,15 @@ class TaskController {
       executionRepository.retrievePipelinesForPipelineConfigId(it, executionCriteria)
     }).subscribeOn(Schedulers.io()).toList().toBlocking().single().sort(startTimeOrId)
 
+    if (!expand) {
+      allPipelines.each { pipeline ->
+        pipeline.getStages().each { stage ->
```

Comment:
Should we filter on the stage type? Just to make sure this only hits the run jobs stages?

Reply:
great idea

Reply:
done